### PR TITLE
Move Catalog data into database, faster control select autocomplete

### PR DESCRIPTION
**Pull request by gregelin**
Move control catalog data into database so catalogs shipped with GovReady and user added catalogs all accessed in database. Also faster reading in containers.

Adjust first_run to load catalogs.

Speed up autocomplete by only selecting matching controls.

### CODE REVIEW

File: controls/models.py [hunk 1]
```diff
@@ -796,44 +794,18 @@ def __init__(self):
...
     def _list_keys(self):
-        return self.extend_external_baselines([
-            'NIST_SP-800-53_rev4',
-            # 'NIST_SP-800-53_rev5',
-            'NIST_SP-800-171_rev1',
-            'CMMC_ver1'
-        ], "keys")
-
+        # TODO: only return keys for records that have baselines?
+        return [item['catalog_key'] for item in CatalogData.objects.order_by().values('catalog_key').distinct()]
```
Comment (alexanderward) [inline controls/models.py:799]: from [item['catalog_key'] for item in CatalogData.objects.order_by().values('catalog_key').distinct()

to

CatalogData.objects.order_by('catalog_key').values_list('catalog_key', flat=True).distinct()

File: controls/oscal.py [hunk 2]
```diff
@@ -6,56 +6,47 @@
...
-
     def _list_catalog_keys(self):
-
-        return self.extend_external_catalogs([
-            Catalogs.NIST_SP_800_53_rev4,
-            Catalogs.NIST_SP_800_53_rev5,
-            Catalogs.NIST_SP_800_171_rev1,
-            Catalogs.CMMC_ver1
-        ], "keys")
+        return [item['catalog_key'] for item in CatalogData.objects.order_by().values('catalog_key').distinct()]
```
Comment (alexanderward) [inline controls/oscal.py:46]: from [item['catalog_key'] for item in CatalogData.objects.order_by().values('catalog_key').distinct()

to

CatalogData.objects.order_by('catalog_key').values_list('catalog_key', flat=True).distinct()

**Review by alexanderward — CHANGES_REQUESTED** [review]

File: controls/models.py (hunk 1 above)
Comment (gregelin) [reply controls/models.py:798]: Change made.

File: controls/oscal.py (hunk 2 above)
Comment (gregelin) [reply controls/oscal.py:46]: Change made.

**Review by alexanderward — APPROVED** [review]

**Pull request merged**